examples/light-switch-app/nrfconnect/README.md

```diff
@@ -577,7 +583,35 @@ To perform binding process you need to complete following steps:
     __{"fabricIndex": 1, "node": <1>, "endpoint": 1, "cluster": 8}__ is a binding for __LevelControl__ cluster. 
 
 
-To test communication between Light Switch and bound device use [buttons](#buttons) or [CLI commands](#matter-cli-commands). Both of them are described in [Device UI section](#device-ui).
+### Binding multicast devices using chip-tool for Windows/Linux
+Binding multicast allows to add all lighting devices to one multicast group. After that Light Switch can send multicast request and all of devices listening bound multicast group can run received command. It can be used to control more than one lighting device via single light switch at the same time.
```

Comment:
In the Matter spec it is not called multicast, but groupcast communication, so I would change it to the "Groupcast binding". Also binding doesn't allow to add all light bulbs to one group. That is done by the modification of group table using groups cluster. Binding only allows to create one to many relationship, so to control whole group at once.